community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/resume.py

```diff
@@ -101,8 +101,16 @@ def instance_properties(nodeset, model, placement_group, labels=None):
     # Override with properties explicit specified in the nodeset
     props.update(nodeset.get("instance_properties") or {})
     
+    if (nodeset.dws_flex.enabled):
```

Comment:
It should be called before line 101. `nodeset.instance_properties` should override whatever our scripts came up with.